facilitator/src/transport/s3.rs

```diff
@@ -192,13 +198,33 @@ impl Transport for S3Transport {
         info!("get {}/{} as {:?}", self.path, key, self.iam_role);
         let mut runtime = basic_runtime()?;
         let client = (self.client_provider)(&self.path.region, self.iam_role.clone())?;
-        let get_output = runtime
-            .block_on(client.get_object(GetObjectRequest {
+
+        let mut retries = 0;
+        let get_output = loop {
+            match runtime.block_on(client.get_object(GetObjectRequest {
                 bucket: self.path.bucket.to_owned(),
                 key: [&self.path.key, key].concat(),
                 ..Default::default()
-            }))
-            .context("error getting S3 object")?;
+            })) {
+                Ok(get_output) => break Ok(get_output),
+                Err(RusotoError::HttpDispatch(err)) => {
+                    retries += 1;
+                    if retries == ATTEMPT_COUNT {
```

Comment:
```suggestion
                    if retries >= ATTEMPT_COUNT {
```

Always good to be paranoid just in case you somehow increment twice.